### ISSUE TITLE
Build error: Opm::EclipseGridInspector::EclipseGridInspector

### ISSUE THREAD

**Issue by joakim-hove** [OPM/opm-upscaling#174]
Fallout after: https://github.com/OPM/dune-cornerpoint/pull/207??

```txt
Opm::EclipseGridInspector::EclipseGridInspector(std::shared_ptr<Opm::Deck const>)'
/private/jenkins/jenkins/OPM/install/RH6/include/opm/porsol/common/ReservoirPropertyCommon_impl.hpp:766: undefined reference to `Opm::EclipseGridInspector::gridSize() const'
```

**Comment by atgeirr** [comment]
No, from the opm-output change. It seems upscaling (and porsol) was ignored in testing for that. I think perhaps the best is to first do the porsol -> upscaling merge, and then fix this.

**Comment by bska** [comment]
> I think perhaps the best is to first do the porsol -> upscaling merge, and then fix this.

Agreed.  We also need to list opm-output as `REQUIRED` in [opm-upscaling-prereqs.cmake](https://github.com/OPM/opm-common/blob/master/cmake/Modules/opm-upscaling-prereqs.cmake) because opm-upscaling (and -porsol) are littered with uses of `opm/`~~`core/io`~~`output/eclipse` headers and need to link to appropriate libraries.

**Comment by atgeirr** [comment]
This has been fixed as of #175 and OPM/opm-common#100. Closing.
